closed/openj9_version_info.h.in

```diff
@@ -36,7 +36,7 @@
 #define OPENJ9_TAG                "@OPENJ9_TAG@"
 #define J9JDK_EXT_VERSION         "@J9JDK_EXT_VERSION@"
 #define J9JDK_EXT_NAME            "@J9JDK_EXT_NAME@"
-#define VENDOR_SHA                "@CLOSEDJ9_SHA@"
+#define VENDOR_SHA                "@VENDOR_SHA@"
```

Comment:
Should I move this after the next line?

Reply:
This list of names was not already sorted, but if you want to sort it, that's fine with me.

Reply:
Sorted them.